me.gladwell.eclipse.m2e.android/plugin.xml

```diff
@@ -1,51 +1,54 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
That is not needed i think. Simply every committer should follow the [convention](https://help.github.com/articles/dealing-with-line-endings/) and use unix delimiters. Than git will change all line endings to unix style (like it did in this case).

Maybe you should change all line-endings in a separate commit for consistency.